assessment-api/open-api-specification.yml

```diff
@@ -24,8 +41,54 @@ paths:
           description: 'Not found'
         '500':
           description: 'Internal server error'
+  /assessments/{assessment-id}:
+    get:
+      tags:
+        - 'assessment'
+      summary: 'get assessment by id '
+      operationId: 'getAssessment'
+      parameters:
+        - name: 'assessment-id'
+          in: 'path'
+          required: true
+          schema:
+            type: 'integer'
+            format: 'int64'
+            example: '1234567890'
+      responses:
+        '200':
+          description: 'Successful operation'
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/assessmentDetail"
+        '400':
+          description: 'Bad request'
+        '401':
+          description: 'Unauthorized'
+        '404':
+          description: 'Not found'
+        '500':
+          description: 'Internal server error'
 
 components:
+  parameters:
+    assessmentParameter:
+      name: name
+      in: query
+      schema:
+        type: string
+        enum:
+          - 'billingAssessment'
```

Comment:
Is it possible to drop the 'Assessment' bit from these enum values, and make them more standard?
AssessmentType.BILLING
AssessmentType.POA
AssessmentType.POA_PREPOP

Reply:
these are the existing values for the assessment names, as much as i would agree this is what OPA is currently using and looking up through the connector. Once we have amended the connector to use this as well, we could easily amend this